flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -443,6 +443,36 @@ public class ExecutionConfigOptions {
                             "The max number of async retry attempts to make before task "
                                     + "execution is failed.");
 
+    // ------------------------------------------------------------------------
+    //  Async ML_PREDICT Options
+    // ------------------------------------------------------------------------
+
+    @Documentation.TableOption(execMode = Documentation.ExecMode.BATCH_STREAMING)
+    public static final ConfigOption<Integer> TABLE_EXEC_ASYNC_ML_PREDICT_BUFFER_CAPACITY =
+            key("table.exec.async-ml-predict.buffer-capacity")
+                    .intType()
+                    .defaultValue(100)
+                    .withDescription(
+                            "The max number of async i/o operation that the async ml predict can trigger.");
```

Comment:
Can we say in the description what happens when the limit is reached? And maybe some advice on how to set this 

Also the configuration name indicates this relates to buffer capacity, but the description is talking about the maximum number of operations.  

If this is buffer capacity what is the units of the 100